From d2d37507b802d34f522dd838e59e4a25ff808158 Mon Sep 17 00:00:00 2001
From: Loc Pham <loc.pham.xh@renesas.com>
Date: Fri, 24 May 2019 13:08:21 +0700
Subject: [PATCH 2/4] arm64: defconfig: Built-in for USB Type C

This patch will change from module to built-in kernel for USB TypeC and
USB TypeC HD3SS3220.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/configs/defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 099d1ac..2a10fc4 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -757,7 +757,7 @@ CONFIG_DRM_I2C_NXP_TDA998X=y
 CONFIG_SND_SOC_ALL_CODECS=y
 CONFIG_COMPILE_TEST=y
 
-CONFIG_USB_TYPEC=m
-CONFIG_TYPEC_HD3SS3220=m
+CONFIG_USB_TYPEC=y
+CONFIG_TYPEC_HD3SS3220=y
 CONFIG_DRM_PANEL_LVDS=y
 CONFIG_COMMON_CLK_VC5=y
-- 
2.7.4

